build.gradle.kts

```diff
@@ -113,6 +113,11 @@ tasks {
     withType<Test> {
         useJUnitPlatform()
     }
+    withType<org.akhikhl.gretty.AppBeforeIntegrationTestTask> {
+        doFirst {
+            jvmArgs = listOf("-DSMTP_PORT=3025", "-DLDAP_URL=ldap://localhost:3389")
```

Comment:
only override these on integration tests and not always so you can run ldap / mail locally without conflicts on tests